app/src/main/res/values/strings.xml

```diff
@@ -1001,8 +1001,8 @@
     <string name="pref_log_line_period_error">Please enter an integer between 1 and 1000.</string>
     <string name="pref_log_write_period_summary">When recording, write to SD card every %1$d lines.</string>
     <string name="pref_log_write_period_title">Write Period</string>
-    <string name="pref_show_timestamp_summary">Show process id and timestamp when expanded.</string>
-    <string name="pref_show_timestamp_title">Show Pid &amp; Timestamp</string>
+    <string name="pref_show_timestamp_summary">Show process ID and timestamp when expanded.</string>
```

Comment:
What is this the timestamp of? Explanation no clearer than the title in that regard.